covid19.py

```diff
@@ -1,38 +1,41 @@
-import requests
-import pandas as pd
 from datetime import datetime
 
+import requests
```

Comment:
can you add a `requirements.txt` file to this project please?

Reply:
Done! No need to use requests library at all because pandas.read_csv() takes care of that.